server/src/main/java/org/elasticsearch/client/transport/TransportClient.java

```diff
@@ -191,6 +191,7 @@ private static ClientTemplate buildTemplate(Settings providedSettings, Settings
             modules.add(actionModule);
 
             CircuitBreakerService circuitBreakerService = Node.createCircuitBreakerService(settingsModule.getSettings(),
+                Collections.emptyList(),
```

Comment:
@rjernst Should we gather the circuit breaker plugin values here as well?

Reply:
I'm surprised we even construct the circuit breaker service. We shouldnt need plugins circuit breakers here.